pkgs/development/tools/marksman/default.nix

```
@@ -0,0 +1,63 @@
+{ lib
+, fetchFromGitHub
+, buildDotnetModule
+, dotnetCorePackages
+, marksman
+, testVersion
+}:
+
+buildDotnetModule rec {
+  pname = "marksman";
+  releaseTag = "2022-10-30";
+  version = "unstable-${releaseTag}";
```

```suggestion
  version = "2022-10-30";
```
there was a confusion here. I thought this was an unstable tag.